spec/features/views_a_drink_spec.rb

```diff
@@ -31,6 +28,7 @@
 
     expect(page).to have_content drink.user.username
     expect(page).not_to have_content "Edit Drink"
+    expect(find('img')['src']).to have_content "default.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.